sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1529,7 +1559,35 @@ static void _send(gnrc_netif_t *netif, gnrc_pktsnip_t *pkt, bool push_back)
      * layer implementations in case `gnrc_netif_pktq` is included */
     gnrc_pktbuf_hold(pkt, 1);
 #endif /* IS_USED(MODULE_GNRC_NETIF_PKTQ) */
+
+#if IS_USED(MODULE_NETSTATS_NEIGHBOR)
+    gnrc_netif_hdr_t *netif_hdr = pkt->data;
+    if (netif_hdr->flags &
+            (GNRC_NETIF_HDR_FLAGS_BROADCAST | GNRC_NETIF_HDR_FLAGS_MULTICAST)) {
+        DEBUG("l2 stats: Destination is multicast or unicast, NULL recorded\n");
+        netstats_nb_record(&netif->netif, NULL, 0);
+    } else {
+        DEBUG("l2 stats: recording transmission\n");
+        netstats_nb_record(&netif->netif,
+                           gnrc_netif_hdr_get_dst_addr(netif_hdr),
+                           netif_hdr->dst_l2addr_len);
+    }
+#endif
+
     res = netif->ops->send(netif, pkt);
+    if (res < 0) {
+        DEBUG("gnrc_netif: error sending packet %p (code: %i)\n",
+              (void *)pkt, res);
+
+#if IS_USED(MODULE_NETSTATS_NEIGHBOR)
+        netstats_nb_update_tx(&netif->netif, NETSTATS_NB_BUSY, 0);
+#endif
+    }
+#ifdef MODULE_NETSTATS_L2
+    else {
+        netif->stats.tx_bytes += res;
+    }
+#endif
```

Comment:
Why is this added? Merge conflict resolve error maybe?

Reply:
Moved from [here](https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/netif/gnrc_netif.c#L1576)

Reply:
Mhh not sure, why I did not see the removal before. Nvm.